core/controllers/creator_dashboard_test.py

```diff
@@ -726,3 +822,36 @@ def test_new_exploration_ids(self):
         self.assertEqual(len(exp_a_id), 12)
 
         self.logout()
+
+    def test_can_upload_exploration(self):
+        with self.swap(constants, 'ALLOW_YAML_FILE_UPLOAD', True):
+            self.set_admins([self.ADMIN_USERNAME])
+            self.login(self.ADMIN_EMAIL, is_super_admin=True)
+
+            response = self.get_html_response(feconf.CREATOR_DASHBOARD_URL)
+            csrf_token = self.get_csrf_token_from_response(response)
+
+            yaml_path = os.path.join(
+                feconf.TESTS_DATA_DIR, 'string_classifier_test.yaml')
+            with open(yaml_path, 'r') as yaml_file:
+                yaml_content = yaml_file.read()
+                self.post_json(
+                    feconf.UPLOAD_EXPLORATION_URL, {
+                        'yaml_file': yaml_content}, csrf_token=csrf_token)
+            self.logout()
+
+    def test_can_not_upload_exploration(self):
```

Comment:
These names aren't descriptive enough. Please follow the guidelines liked above.

Reply:
Is it OK now?